spec/syntax/sigil_spec.rb

```diff
@@ -87,6 +87,11 @@
     it 'without escaped parenthesis' do
       expect('~S(\( )').not_to include_elixir_syntax('elixirRegexEscapePunctuation', '( ')
     end
+
+    it 'Live EEx' do
+      expect('~L"""liveview template"""').to include_elixir_syntax('elixirSigilDelimiter', '""""')
+      expect('~L"""liveview template"""').to include_elixir_syntax('elixirSigilDelimiter', '""""')
```

Comment:
Are these expects different? Could we just have one?

Reply:
If possible it would be nice to get a test on the inner sigil syntax being recognized. I can see if I can get one written

Reply:
I removed the duplicated line, but haven't got time to write test for inner sigil syntax yet.